app/server/pkg/community/github/analytics.go

```diff
@@ -0,0 +1,93 @@
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	timeInterval = 1 * time.Hour
+	repoName     = "litmus"
+)
+
+type GithubData struct{
+	Stars string `json:"stars"`
+	ExperimentCount string `json:"experiment-count"`
+}
+
+var Github GithubData
+
+// Handler is responsible for the looping the UpdateGithubData()
+func Handler() {
+	for true {
+		log.Infof("Updating Github Litmus Repo Data ...")
+		err := UpdateGithubStars()
```

Comment:
If you are not accessing the method outside make the method name start with small letter.